packages/bundler-sdk-viem/src/BundlerAction.ts

```diff
@@ -707,6 +712,40 @@ export namespace BundlerAction {
     ];
   }
 
+  /**
+   * Encodes a call to the GeneralAdapter1 to wrap legacy MORPHO tokens.
+   * @param chainId The chain id for which to encode the call.
+   * @param recipient The recipient of MORPHO tokens.
+   * @param amount The amount of tokens to wrap.
+   * @param skipRevert Whether to allow the wrap to revert without making the whole bundler revert. Defaults to false.
+   */
+  export function morphoWrapperDepositFor(
+    chainId: ChainId,
+    recipient: Address,
+    amount: bigint,
+    skipRevert = false,
+  ): BundlerCall[] {
+    if (chainId !== ChainId.EthMainnet)
+      throw new Error("MORPHO wrapping is only available on ethereum mainnet");
+    const {
+      bundler3: { generalAdapter1 },
+    } = getChainAddresses(chainId);
```

Comment:
`morphoWrapperWithdrawTo` is missing, but maybe never used?

Reply:
I thought that since it's not likely to be used (is there a use-case where we wanna retrieve legacy tokens?) we better not expose it to avoid confusion and useless maintainance

Reply:
wdyt?